Tests/Easy-Purchase-Receipt-ValidatorTests/ASN1/ASN1ObjectTest.swift

```
@@ -68,13 +69,13 @@ final class ASN1ObjectTest: XCTestCase {
     0x01 0x01 0xFF   ; Boolean (TRUE)
     0x06, 0x09, 0x60, 0x86, 0x48, 0x1, 0x65, 0x3, 0x4, 0x2, 0x1, OID (sha256 = "2.16.840.1.101.3.4.2.1")
  */
-extension ASN1ObjectTest{
+extension ASN1ObjectTest {
     func initialSetup(with data: [UInt8]) {
         let decoder = ASN1Decoder()
         let data = Data(data)
         do {
             let asn1ObjectList = try decoder.decode(data: data)
             self.sut = asn1ObjectList.first
-        }catch _ {}
+        } catch _ {}
```

empty catch block? can we do some implementation on catch block?

---

I can forcefully do try here but it make lint error. So this way. It's in test file that's why not handling catch.